Backend/api/choicesapi.py

```diff
@@ -17,68 +21,91 @@ def read_choices(electionid: int) -> str:
                 'counter': choice.counter
             }
         )
-    return json.dumps(liste)
+    return Response.ok(json.dumps(liste))
 
 def create_choice(data: dict) -> str:
     if not 'description' in data:
-        return json.dumps({'message':'description missing'})
+        return Response.wrong_format(json.dumps({'message':'description missing'}))
+    if not 'electionid' in data:
+        return Response.wrong_format(json.dumps({'message':'electionid missing'}))
+
+    if 'picture' in data:
+        if data['picture'].endswith('=='):
+            picture = data['picture']
+        else:
+            return Response.wrong_format({"message" : "picture is not Base64"})
+    else :
+        picture = ''
 
-    picture = data['picture'] if ( 
-        'picture' in data and data['picture'].endswith('==')
-        ) else ''
-    
     choice = Choice(
         description=data['description'], 
         counter=0, 
         picture=picture,
-        election_round_id=data['electionId']
+        election_round_id=data['electionid']

```

Comment:
um model und response gleich zu halten wäre election_round_id als paramter besser 
```suggestion
        election_round_id=data['election_round_id']

```